rust-lib/Cargo.toml

```diff
@@ -39,5 +41,3 @@ features = ["bundled"]
 
 [target.'cfg(not(target_os="android"))'.dependencies]
 env_logger = "~0.7.1"
-
-[workspace]
```

Comment:
Why we remove this? It is needed for dependency tests

Reply:
I erroneously assumed it was no necessary, sorry